EventKit/eventkit-cloud
# Users filter bug

This PR:
- Fixes bug when filtering the members/group based off of permissions and group/member name that was configured/changed in the backend
- Pulled in backend changes from Svrcek's branch to enable ordering for frontend
- Updated api calls in redux actions for users/members and groups
- Updated all function calls necessary to point to the dispatched action for the api call where state is also updated
- Removed any unnecessary frontend filtering
- Modified old tests to reflect current/updated behavior

To Test:
- Select a completed datapack
- Select that datapack's shared permissions
- Check ordering is correct for different scenarios -admin, not admin, shared, non shared
- If datapack user is not selected, ensure user is not seen inside the ordered results ('exclude_self' was added to params for this purpose)
- Ensure an api call is made for every filter, reverse ordering, loading more, search input
- Ensure url includes job uid, 'exclude_self=true', ordering=shared/admin_shared,name/username, and page number
